src/scripting_api/app.js

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { EventDispatcher } from "./event.js";
+import { NotSupportedError } from "./error.js";
+import { PDFObject } from "./pdf_object.js";
+
+class App extends PDFObject {
+  constructor(data) {
+    super(data);
+    this._specialId = true;
+    this._document = data._document;
+    this._setTimeout = setTimeout;
+    this._clearTimeout = clearTimeout;
+    this._setInterval = setInterval;
+    this._clearInterval = clearInterval;
+    this._calculationOrder = data.calculationOrder;
```

Comment:
This member is not used anywhere, so I think it can removed. The calculation order is inserted below directly.